01-JS-DOM-APIs/js/First.js

```diff
@@ -0,0 +1,65 @@
+function messageFirst() {
+  setTimeout(function () {
+    document.getElementById("content").innerHTML = "Hello world";
+  }, 2000);
+}
+
+function buttonClick(){
+  let config = {
+    url: 'http://api.icndb.com/jokes/random'
+  }
+  function resolve(xhttp) {
```

Comment:
Why you have this parameter if you are not going to use it? Here you should use the response you are going to use. In this case  `event`